indico/modules/events/abstracts/controllers/management.py

```diff
@@ -123,3 +142,79 @@ class RHAbstractListStaticURL(RHAbstractListBase):
 
     def _process(self):
         return jsonify(url=self.list_generator.generate_static_url())
+
+
+class RHCreateAbstract(RHAbstractListBase):
+    def _process(self):
+        abstract_form_class = make_abstract_form(self.event_new)
+        form = abstract_form_class(event=self.event_new)
+        if form.validate_on_submit():
+            data = form.data
+            if 'attachments' in data:
```

Comment:
fyi, you can `data.pop('attachments', None)` to remove it if it's present and not do anything if not